modules/ROOT/pages/indexes/search-performance-indexes/list-indexes.adoc

```diff
@@ -255,7 +247,7 @@ label:default-output[]
 
 | `options`
 | Information retrieved from the `OPTIONS` map about the provider and configuration settings for an index.
```

Comment:
In Cypher 5 we return both provider and configuration settings but in Cypher 25 we only return the configuration settings in the options column. So this sentence should maybe be updated to reflect that 🤔 This would also be true for show constraints as well and not just show index (as I assume it has a similar sentence around it's options column).

Not really related to your PR I just noticed this sentence now.